src/components/typography/IconLink.js

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { Link } from 'gatsby';
+
+const IconLink = ({ url, style, icon, alt }) => {
+    return (
+        <Link to={url} target="_blank">
```

Comment:
Jeśli korzystamy z target="_blank" powinniśmy też dodać odpowiedni atrybut: `rel="noopener noreferrer"`

https://stackoverflow.com/questions/50709625/link-with-target-blank-and-rel-noopener-noreferrer-still-vulnerable